src/components/InternshipEngagementListItemCard/InternshipEngagementListItem.graphql

```diff
@@ -32,4 +32,13 @@ fragment InternshipEngagementListItem on InternshipEngagement {
   completeDate {
     value
   }
+  project {
+    marketingCountry {
+      canRead
+      canEdit
+      value {
+        ...DisplayLocation
+      }
+    }
+  }
```

Comment:
I think this was the right place to put this.  Adding this fixed a bug so that when a languages sensitivity is changed the marketingCountry DataButton will update to show either the Region or the Country based on its sensitivity.